agoradatatools/etl/load.py

```diff
@@ -17,21 +17,21 @@ def default(self, obj):
         return json.JSONEncoder.default(self, obj)
 
 
-def create_temp_location():
+def create_temp_location(staging_path: str):
```

Comment:
I did see this and I had planned on @BWMac to eventually use the tempfile module in python to create these temp locations.

That said, thanks for contributing this refactor.  This is an excellent improvement from what was being done.